olp-cpp-sdk-core/tests/thread/ExecutionContextTest.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2022 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <chrono>
+#include <future>
+#include <thread>
+
+#include <gtest/gtest.h>
+
+#include <olp/core/thread/ExecutionContext.h>
+
+class ExecutionContextTest : public testing::Test {
+ protected:
+  olp::thread::ExecutionContext execution_context;
+};
```

Comment:
Do you really need this or can you simply use TEST() and declare the `ExecutionContext` in every test?
I would not use a fixture.

Reply:
done